satellite/repair/checker/checker.go

```diff
@@ -293,6 +296,14 @@ func (obs *checkerObserver) RemoteSegment(ctx context.Context, path metainfo.Sco
 			}
 		}
 
+		var segmentAge time.Duration
+		if pointer.CreationDate.Before(pointer.LastRepaired) {
+			segmentAge = time.Since(pointer.CreationDate)
```

Comment:
this is flipped - if creation date is before last repaired, we want to do `time.Since(lastRepaired)`. If creation date is later, it means `lastRepaired` hasn't been set yet so we want to use `CreationDate`

Reply:
gah sorry about that